tox.ini

```diff
@@ -130,6 +130,7 @@ markers =
     ingress: tag a BDD feature as related to ingress
     volume: tag a BDD feature as related to Volume management
     bootstrap: tag a BDD feature as related to bootstrap
+    authentication: tag a BDD feature as related to authentication
```

Comment:
Good point, however this should have its dedicated commit (not related to your changes above).